renderlib/ScaleBarTool.cpp

```diff
@@ -0,0 +1,125 @@
+#include "ScaleBarTool.h"
+
+#include "ImageXYZC.h"
+#include "Logging.h"
+
+#include <iomanip>
+#include <sstream>
+
+// physicalscale is max of physical dims x,y,z
+static float
+computePhysicalScaleBarSize(const float physicalScale)
+{
+  // note this result will always be some integer power of 10 independent of zoom...
+  return pow(10.0f, floor(log10(physicalScale / 2.0f)));
+}
```

Comment:
What does dividing by 2 here do?

Reply:
good question.  I think I saw it in volume viewer and thought it was just giving some rounding-down effect to the result value.  I'll test with and without